### PR TITLE
Grammatical Tweaks (Section 05-06)

### PR DESCRIPTION
**Pull request by furey**
Following https://github.com/adonisjs/docs/pull/314, this PR applies numerous grammatical tweaks to sections 05-06 of the documentation.

### CODE REVIEW

File: 05-Security/02-Authentication.adoc
Comment (RomainLanz) [inline 05-Security/02-Authentication.adoc]: We should remove the term `REST` here.
```suggestion

Using authenticators you can build traditional *session based* login systems and secure your *APIs*.
```

File: 05-Security/01-Getting-Started.adoc
Comment (RomainLanz) [inline 05-Security/01-Getting-Started.adoc]: Let's remove the `REST` term here.

```suggestion
As HTML forms are only capable of making `GET` and `POST` requests, you cannot use HTTP verbs like `PUT` or `DELETE` to perform resourceful operations via a form's `method` attribute.
```

File: 05-Security/02-Authentication.adoc
Comment (RomainLanz) [inline 05-Security/02-Authentication.adoc]: Also here 

```suggestion
*Stateless* authentication requires the user to resend their credentials on each HTTP request, which is very common with APIs.
```

File: 05-Security/02-Authentication.adoc
Comment (RomainLanz) [inline 05-Security/02-Authentication.adoc]: There's no need to extract the `response` from the context here.
Also, could be good to keep them in the alphabetical order.

```suggestion
  async login ({ auth, request }) {
```

**Review by RomainLanz — CHANGES_REQUESTED** [review]
Wow! Thanks again for this PR @furey! 😍 

I believe we can also add a link to [Adonis Persona](https://github.com/adonisjs/adonis-persona) in the authentication section as a `note` or `tips`.

**Comment by furey** [comment]
> I believe we can also add a link to Adonis Persona in the authentication section as a note or tips.

✅ https://github.com/adonisjs/docs/pull/328/commits/7b6a76bc53250cc1e01081259eeec5d9a5dd611b

**Review by furey — COMMENTED** [review]
> Wow! Thanks again for this PR @furey! 😍

No worries!

**Review by RomainLanz — APPROVED** [review]

**Comment by furey** [comment]
Hi @RomainLanz & @thetutlage,

I'd finished parsing the "06-Digging-Deeper" section of the documentation, so thought I'd update this PR with those files while still waiting for approval/merge.

Cheers!

**Comment by thetutlage** [comment]
@furey Do you want us to wait for the other changes, or should I go ahead and merge it right now and you will do a different PR for rest of the changes?

**Comment by furey** [comment]
@thetutlage Happy for you to merge now and I'll submit a different PR for the rest. Thanks! 👍

**Comment by RomainLanz** [comment]
Thanks a lot!

**Pull request merged**
